frontend/package-lock.json

```diff
@@ -6402,7 +6402,8 @@
     "@types/js-yaml": {
       "version": "3.11.4",
       "resolved": "https://registry.npmjs.org/@types/js-yaml/-/js-yaml-3.11.4.tgz",
-      "integrity": "sha512-a42nbZzOlxbH3ORDg/brU+zlRQS8mSvZCEY4Xery66/NIA7yX4T9qk6yL9qpw4AhgwzEgc1XIhHRnivQW+zlkw=="
+      "integrity": "sha512-a42nbZzOlxbH3ORDg/brU+zlRQS8mSvZCEY4Xery66/NIA7yX4T9qk6yL9qpw4AhgwzEgc1XIhHRnivQW+zlkw==",
```

Comment:
actually i removed `@types/js-yaml` from package.json cuz i don't see it used anywhere. But I reverted it anyway.

And we probably want to put in in dev, since everything is compiled to js.